src/server/auth/cmds/cmds_test.go

```diff
@@ -80,12 +79,12 @@ func TestAuthBasic(t *testing.T) {
 	activateAuth(t)
 	defer deactivateAuth(t)
 	require.NoError(t, tu.BashCmd(`
-		echo "{{.alice}}" | pachctl auth login
-		pachctl create repo {{.repo}}
-		pachctl list repo \
-			| match {{.repo}}
-		pachctl inspect repo {{.repo}}
-		`,
+			echo "{{.alice}}" | pachctl auth login
```

Comment:
I don't really object but why the extra spacing? Should we be doing this in general?